README.md

```diff
@@ -76,6 +76,31 @@ Results in:
 <script src="/js/jquery-[hash].js" async></script>
 ```
 
+You can also reference image paths via the `assetPath` helper. First, you must specify the
+path to your images via the `paths` option e.g:
+```javascript
+...
+
+var assets = require('connect-assets');
+
+app.use(assets({
+  paths: [
+    'assets/css',
+    'assets/js',
+    'assets/img'
+  ]
+}));
+```
+You can then use the `assetPath` helper in your Jade like so:
+```
+img(src="#{assetPath('image-name.png')}") 
```

Comment:
Oops, I did not think to do it without the interpolation (I am just getting started with node/jade/express/etc.), and I think it looks cleaner without after seeing it.  Either way, I can do another pull request with the changes if you'd like.